gallery/src/styles/NxVulnerabilityScore/NxVulnerabilityScoreTableExample.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { NxVulnerabilityIndicator, formatVulnerabilityScore } from '@sonatype/react-shared-components';
+
+const NxVulnerabilityNumberTableExample = () =>
```

Comment:
```suggestion
const NxVulnerabilityScoreTableExample = () =>
```

Reply:
Sorry about that, fixed here: [f460489](https://github.com/sonatype/sonatype-react-shared-components/pull/1199/commits/f460489a77bb351c6e936b902dc3f9a1e212de74)